examples/full-example/app.js

```diff
@@ -47,4 +47,27 @@ function bundle(req, bundle, variations) {
     return '<script src="'+url+'"></script>';
 }
 
+function entryMap(req, bundle, variations) {
+    // req.mendel.getBundleEntries contains all bundles as keys and array of
+    // entries that were used and normalized by variations, this allows apps
+    // to do their specific logic with their bundles
+    var bundles = req.mendel.getBundleEntries();
```

Comment:
We can, this won't save any performance since this function executes in nanoseconds, I put it here because it reads well for people learning to have this be grouped with the next command. Also makes the request code above simpler and the params exactly the same as the `bundle` function.